rotkehlchen/db/optimismtx.py

```diff
@@ -0,0 +1,74 @@
+import logging
+from typing import TYPE_CHECKING, Any, Optional
+
+from rotkehlchen.chain.optimism.types import OptimismTransaction
+from rotkehlchen.db.evmtx import DBEvmTx
+from rotkehlchen.logging import RotkehlchenLogsAdapter
+from rotkehlchen.serialization.deserialize import deserialize_timestamp
+from rotkehlchen.types import ChainID, ChecksumEvmAddress, deserialize_evm_tx_hash
+
+if TYPE_CHECKING:
+    from rotkehlchen.db.drivers.gevent import DBCursor
+
+logger = logging.getLogger(__name__)
+log = RotkehlchenLogsAdapter(logger)
+
+
+class DBOptimismTx(DBEvmTx):
+
+    def add_evm_transactions(
+            self,
+            write_cursor: 'DBCursor',
+            evm_transactions: list[OptimismTransaction],  # type: ignore[override]
+            relevant_address: Optional[ChecksumEvmAddress],
+    ) -> None:
+        """Adds optimism transactions to the database"""
+        super().add_evm_transactions(
+            write_cursor,
+            evm_transactions,  # type: ignore[arg-type]
+            relevant_address,
+        )
+
+        tx_tuples: list[tuple[Any, ...]] = []
+        for tx in evm_transactions:
+            tx_tuples.append((
+                tx.tx_hash,
+                tx.l1_fee,
+            ))
+
+        query = """
+            INSERT INTO optimism_transactions(
+              tx_hash,
+              l1_fee)
+            VALUES (?, ?)
+        """
+        self.db.write_tuples(
+            write_cursor=write_cursor,
+            tuple_type='evm_transaction',
+            query=query,
+            tuples=tx_tuples,
+            relevant_address=relevant_address,
```

Comment:
Remove relevant address from here as the logic concerning it should already be handled when you add the main data.
The only point of using the `write_tuples` function and not doing just `executemany` directly is to repeat with a for loop if one transaction already exists in the DB.